readme.md

```diff
@@ -91,6 +91,16 @@ yarn release --release-as 1.1.0
 
 In general, you can pass any flag supported by standard-version, please refer to their [CLI usage](https://github.com/conventional-changelog/standard-version#cli-usage) documentation.
 
+**Publish a github release**
```

Comment:
Same. If the `release-it` command is going to replace `standard-version`, we can remove the previous section!

Reply:
see above